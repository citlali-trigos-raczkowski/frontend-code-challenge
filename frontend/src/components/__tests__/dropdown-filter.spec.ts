import { shallowMount, createLocalVue } from "@vue/test-utils";
import DropdownFilter from "../nav-components/dropdown-filter.vue";
import VueRouter from "vue-router";
import { DropdownFilterMockData } from "../../../tests/mocks/mock-props";

const localVue = createLocalVue();
localVue.use(VueRouter);

describe("component/dropdown-filter.vue", () => {
  it("renders correctly", () => {
    const wrapper = shallowMount(DropdownFilter, {
      propsData: DropdownFilterMockData,
      localVue,
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
