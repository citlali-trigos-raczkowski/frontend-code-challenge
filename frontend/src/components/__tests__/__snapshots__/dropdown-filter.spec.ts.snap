// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/dropdown-filter.vue renders correctly 1`] = `
<v-select-stub
  appendicon="$dropdown"
  backgroundcolor=""
  clearicon="$clear"
  dark="true"
  data-cy="filter-dropdown"
  dense="true"
  errorcount="1"
  errormessages=""
  filled="true"
  itemcolor="primary"
  itemdisabled="disabled"
  items="fire,water"
  itemtext="text"
  itemvalue="value"
  loaderheight="2"
  menuprops="[object Object]"
  messages=""
  nodatatext="$vuetify.noDataText"
  rounded="true"
  rules=""
  successmessages=""
  type="text"
  valuecomparator="[Function]"
/>
`;
